services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -154,10 +174,27 @@ const getMultisignatureGroups = async account => {
 	return multisignatureAccount;
 };
 
+const indexAccountsbyPublicKey = async (publicKeysToIndex) => {
```

Comment:
It would be good to have the method moved up, close to `indexAccounts` as they're both closely related.